### PR TITLE
[4.0] Missing alerts strings

### PR DESCRIPTION
**Pull request by infograf768**
Pull Request for Issue https://github.com/joomla/joomla-cms/issues/30682

### Summary of Changes
Creating the missing strings in front and backend
JCLOSE="Close"
JOK="OK"
JOPEN="Open"

Added the declarations in message.php

The strings are present in `/media/vendor/joomla-custom-elements/js/joomla-alert-es5.js`

### Testing Instructions
Save an article and look at the markup produced for the X icon closing the alert.

### Actual result BEFORE applying this Pull Request
jclose

### Expected result AFTER applying this Pull Request

Screen Shot 2020-09-19 at 08 57 21

### CODE REVIEW

File: language/en-GB/joomla.ini [hunk 1]
```diff
@@ -65,7 +65,7 @@ JCATEGORY="Category"
 JCATEGORY_ASC="Category ascending"
 JCATEGORY_DESC="Category descending"
 JCLEAR="Clear"
-JDATE="Date"
```
Comment (Bakual) [inline language/en-GB/joomla.ini:68]: Did you remove that one on purpose?

Comment (infograf768) [reply language/en-GB/joomla.ini:68]: nope, correcting now

Comment (infograf768) [reply language/en-GB/joomla.ini:68]: done. thanks

**Comment by ghost** [comment]
I have tested this item :white_check_mark: successfully on d05869cf512d7e402b7254cbb90a27e6e80d2ecc<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/30684">issues.joomla.org/tracker/joomla-cms/30684</a>.</sub>

**Comment by alikon** [comment]
I have tested this item :white_check_mark: successfully on d05869cf512d7e402b7254cbb90a27e6e80d2ecc<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/30684">issues.joomla.org/tracker/joomla-cms/30684</a>.</sub>

**Comment by alikon** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/30684">issues.joomla.org/tracker/joomla-cms/30684</a>.</sub>

**Comment by richard67** [comment]
Thanks!

**Pull request merged**